### PR TITLE
docs: added other language support

### PR DESCRIPTION
**Pull request by medgar-nitric**

### CODE REVIEW

**Review by davemooreuws — APPROVED** [review]
LGTM

File: docs/intro/languages/other.mdx [hunk 1]
```diff
@@ -1 +1,40 @@
-TODO
+Nitric is an open framework designed to support additional languages for multi-cloud development. Languages are supported through SDKs which provide Nitric APIs to cloud services and a FaaS micro-framework to build cloud functions.
+
+If you want add support for another language please contribute to the [GitHub Nitric](https://github.com/nitrictech) open source project. Please reach out to us, we would will be super happy to help.
```
Comment (jyecusch) [inline docs/intro/languages/other.mdx:3]: if you want **_to_** add support...
missing 'to'

File: docs/intro/languages/other.mdx [hunk 2]
```diff
@@ -1 +1,40 @@
...
+| ---------------------------------------------------------------- | -------------------------------------------------------------------- | -------------------------------------------------------------- | ------------------------------------------------------------ |
+| [Node API](https://github.com/nitrictech/apis/tree/develop/node) | [Python API](https://github.com/nitrictech/apis/tree/develop/python) | [JVM API](https://github.com/nitrictech/apis/tree/develop/jvm) | [Go API](https://github.com/nitrictech/apis/tree/develop/go) |
+| [Node SDK](https://github.com/nitrictech/node-sdk)               | [Python SDK](https://github.com/nitrictech/python-sdk)               | [JVM SDK](https://github.com/nitrictech/java-sdk)              | [Go SDK](https://github.com/nitrictech/go-sdk)               |
+
+To add new language support we would recommend you follow the same layering pattern with a generated gRPC base API library and a separate SDK repository which provides a higher level language support developers to use.
```
Comment (jyecusch) [inline docs/intro/languages/other.mdx:29]: Suggest replacing: "which provides a higher level language support developers to use"
With: "which provides higher-level language support for developer use" (or something similar)

File: docs/intro/languages/other.mdx [hunk 3]
```diff
@@ -1 +1,40 @@
...
+
+## Nitric API gRPC Proto Contracts
+
+The [Nitric](https://github.com/nitrictech/nitric) repository maintains the Service API [Proto Contracts](https://github.com/nitrictech/nitric/tree/develop/contracts).
+
+Anytime a Nitric proto contract change is made a GitHub workflow action executes for each lanaguage API which:
+
+1. generates gRPC library code an increments the version
```
Comment (jyecusch) [inline docs/intro/languages/other.mdx:37]: "an" should be "and"
e.g. "and increments the version"

**Review by jyecusch — CHANGES_REQUESTED** [review]
A couple of typos but looks good to me otherwise.

File: docs/intro/languages/other.mdx (hunk 1 above)
Comment (tjholm) [reply docs/intro/languages/other.mdx:3]: also 'we would will be...'

**Review by tjholm — CHANGES_REQUESTED** [review]

**Review by davemooreuws — APPROVED** [review]

**Review by tjholm — COMMENTED** [review]
Content looks good, I think we should probably update the diagram before merging though

**Comment by medgar-nitric** [comment]
> Content looks good, I think we should probably update the diagram before merging though

What changed do you want to the diagram

**Review by tjholm — APPROVED** [review]

**Review by jyecusch — APPROVED** [review]

**Pull request merged**